starter_code/index.js

```diff
@@ -1,3 +1,14 @@
 const Elevator = require('./elevator.js');
+const Person = require("./person.js");
 
+elevator = new Elevator();
```

Comment:
Be careful! 'elevator', 'mark'... are variables, you need to define them with var, let or const.